pallets/ah-migrator/src/lib.rs

```diff
@@ -267,6 +274,16 @@ pub mod pallet {
 			count_bad: u32,
 		},
 		ReferendaProcessed,
+		SchedulerMessagesReceived {
```

Comment:
I added a generic variant with a `PalletName` that we can use as to not copy&paste this always for new pallets.